go-controller/README.md

```diff
@@ -36,6 +36,8 @@ Options specified on the command-line override both defaults and configuration f
 Usage:
   -config-file string
      configuration file path (default: /etc/openvswitch/ovn_k8s.conf)
+  -ha
+     enable ha when start ovnkube which will rebuild ovn db if it is started on a new node
```

Comment:
s/"enable ha when start ovnkube which will rebuild ovn db if it is started on a new node"/" rebuilds ovn db if it is started on a new node"

There is also https://github.com/openvswitch/ovn-kubernetes/blob/master/docs/ovn_k8s.conf.5 that needs updating.

Reply:
I think you are referring to ovnkube.1 instead of ovn_k8s.conf.5